apps/arena/lib/arena/entities.ex

```diff
@@ -36,6 +36,13 @@ defmodule Arena.Entities do
         stamina_interval: character.stamina_interval,
         cooldown_multiplier: 1,
         recharging_stamina: false,
+        max_mana: character.base_mana,
+        mana: 50,
```

Comment:
I think this should be 0 instead of 50 right? is this the current amount of mana?

Reply:
Yes, this is the current mana and it starts at 50

Reply:
nice